interface/modules/custom_modules/oe-module-faxsms/library/api_onetime.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * Portal OneTime for API
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2023 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General public License 3
+ */
+
+require_once(__DIR__ . "/../../../../globals.php");
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Services\PatientPortalService;
+
+if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"], 'contact-form')) {
+    CsrfUtils::csrfNotVerified();
+}
```

Comment:
Does this script need to be protected by an acl?
(or else anybody logged in will be able to run this script and put whatever they want for the $details request)

Reply:
I would think so Brady. However I'm unsure what it should be. We really need new ACLs for portal those patient side and administration side. That's not something for patch so it will probably be admin